src/streamlink/plugins/atpchallenger.py

```diff
@@ -0,0 +1,30 @@
+"""
+$description Tennis tournaments organized by the Association of Tennis Professionals.
+$url atptour.com/en/atp-challenger-tour/challenger-tv
+$type live, vod
+"""
+
+import logging
+import re
+
+from streamlink.plugin import Plugin, pluginmatcher
+from streamlink.plugin.api import validate
+
+log = logging.getLogger(__name__)
```

Comment:
Please remove the logging stuff (import and `log` definition), it's not needed. Forgot to include this in my diff.

Reply:
will do